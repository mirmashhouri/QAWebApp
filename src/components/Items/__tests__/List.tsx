import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";

import List from "..";

describe("<Items />", () => {
  it("renders", () => {
    const tree = render(
      <MemoryRouter>
        <List
          items={[
            {
              id: 1,
              question: "What is your name?",
              answer: "My name is Abbas",
            },
          ]}
        />
      </MemoryRouter>
    ).container.firstChild;

    expect(tree).toMatchSnapshot();
  });
});
