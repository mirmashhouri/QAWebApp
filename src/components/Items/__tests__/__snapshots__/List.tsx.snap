// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Items /> renders 1`] = `
<div
  class="items"
>
  <h4>
    Question List
  </h4>
  <ul>
    <li>
      <a
        href="/UserInfo/1"
      >
        What is your name?
      </a>
      <div>
        My name is Abbas
      </div>
    </li>
  </ul>
</div>
`;
